api.md

```diff
@@ -571,6 +576,16 @@ Drop all chunks from hypertable `conditions` older than 3 months, including depe
 SELECT drop_chunks(interval '3 months', 'conditions', cascade => TRUE);
 ```
 
+Drop all chunks newer than 3 months:
+```sql
+SELECT drop_chunks(newer_than => interval '3 months');
+```
+
+Drop all chunks newer than 3 months and older than 4 months:
```

Comment:
comment is opposite of the command, should read:
`newer than 4 months and older than 3 months`

(as written is an impossible condition :) )

Reply:
whoops - thanks for catching. fixing!